dist/locales/en.json

```diff
@@ -1643,12 +1643,12 @@
                 "title": "Help",
                 "welcome": "Welcome to the iD editor for [OpenHistoricalMap](https://www.openhistoricalmap.org/). With this editor you can update OpenHistoricalMap right from your web browser.",
                 "open_data_h": "Open Data",
-                "open_data": "Edits that you make on this map will be visible to everyone who uses OpenHistoricalMap. Your edits can be based on personal knowledge, on-the-ground surveying, or imagery collected from aerial or street level photos. Copying from commercial sources, like Google Maps, [is strictly forbidden](https://www.openstreetmap.org/copyright).",
+                "open_data": "Edits that you make on this map will be visible to everyone who uses OpenHistoricalMap. Your edits can be based on personal knowledge, on-the-ground surveying, imagery collected from aerial or street level photos, and out-of-copyright maps. Copying from commercial sources, like Google Maps, [is strictly forbidden](https://www.openhistoricalmap.org/copyright).",
```

Comment:
The original message tried to avoid the term copyright, probably because laypeople have a lot of misconceptions about copyright law (only applies when there’s a copyright symbol, can be waived simply by saying “no infringement intended”, etc.). But since this is a historical research project, we probably need a whole chapter on copyright. Maybe we could have it vary by country and present U.S. mappers with the [Hirtle chart](https://commons.wikimedia.org/wiki/Commons:Hirtle_chart).

A responsible mapper would inevitably need to learn the difference between corroborating a fact from a source versus copying from the source, or between a copyrighted or out-of-copyright source. (Pop quiz: when’s [the current cutoff date](https://commons.wikimedia.org/wiki/Template:PD-US-expired) for works other than sound recordings first published in the U.S. to be considered copyright-expired?)